src/pages/merge/ResolutionSelector.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { get } from 'lodash-es';
+
+import Select from '@material-ui/core/Select';
+import FormControl from '@material-ui/core/FormControl';
+import InputLabel from '@material-ui/core/InputLabel';
+import MenuItem from '@material-ui/core/MenuItem';
+
+import sexOptions from '../../constants/sexOptions';
+
+const validSexOptions = sexOptions.filter(o => o.value);
+
+const propertyMap = {
+  sex: {
+    labelId: 'SEX',
+    getProperty: individualData => individualData?.sex,
+    deriveChoices: sexes =>
+      validSexOptions.filter(o => sexes.includes(o.value)),
+  },
+  defaultName: {
+    labelId: 'NAME',
+    getProperty: individualData => {
+      const names = get(individualData, 'names', []);
+      const defaultNameObject = names.find(
+        name => name?.context === 'defaultName',
+      );
+      return defaultNameObject?.value;
+    },
+    deriveChoices: names =>
+      names.map(name => ({
+        label: name,
+        value: name,
+      })),
+  },
+  nickname: {
+    labelId: 'ALIAS', // untested!
+    getProperty: individualData => {
+      const names = get(individualData, 'names', []);
+      const defaultNameObject = names.find(
+        name => name?.context === 'Nickname',
```

Comment:
I think 'Nickname' is supposed to be lower case.

Reply:
nice catch